csharp/src/Microsoft.ML.OnnxRuntime/SessionOptions.cs

```diff
@@ -592,6 +650,31 @@ public ExecutionMode ExecutionMode
         }
         private ExecutionMode _executionMode = ExecutionMode.ORT_SEQUENTIAL;
 
+
+        /// <summary>
+        /// Provider options for TensorRT.
+        /// </summary>
+        /// 
+        //  Example for setting:
+        //    SessionOptions.OrtTensorRTProviderOptions trt_options;
+        //    trt_options.device_id = 0;
+        //    trt_options.has_trt_options = 1;
+        //    trt_options.trt_max_workspace_size = (UIntPtr) (1<<30);
+        //    trt_options.trt_fp16_enable = 1;
+        //    trt_options.trt_int8_enable = 1;
+        //    trt_options.trt_int8_calibration_table_name = "C:\calibration.flatbuffers";
+        //    trt_options.trt_int8_use_native_calibration_table = 0;
+        public struct OrtTensorRTProviderOptions
```

Comment:
somehow the static checking is failing on documentation here:
https://dev.azure.com/onnxruntime/onnxruntime/_build/results?buildId=342807&view=logs&j=72b59a5c-f9fa-5437-adfc-3f855724a45f&t=cd0ac243-b7df-5d25-6746-15e249cb39c3&l=155
Member device_id (variable) of struct Microsoft::ML::OnnxRuntime::SessionOptions::OrtTensorRTProviderOptions is not documented

Reply:
I've fixed this.